worker/diskmanager/lsblk_test.go

```diff
@@ -83,6 +83,15 @@ ID_WWN=foo
 `, storage.BlockDevice{WWN: "foo"})
 }
 
+func (s *ListBlockDevicesSuite) TestListBlockDevicesExtendedWWN(c *gc.C) {
+	// If ID_WWN_WITH_EXTENSION is found, then we should use that
+	// in preference to the ID_WWN value.
+	s.testListBlockDevicesExtended(c, `
+ID_WWN=foo
+ID_WWN_WITH_EXTENSION=foobar
```

Comment:
Might be worth flipping the order of these or adding another test with them in reverse to ensure the with_extension takes priority.

Reply:
indeed